README.md

```diff
@@ -93,3 +93,94 @@ extension BannerView: BannerRendering {
 }
 ```
 
+## 🛰 Interactor
+
+> **Interactor**
+> (*noun*)
+> 
+> Component answering the question: What should I do in reaction to user action. 
+
+### Definitions
+
+### TL;DR
+
+Interactor is a final class that ViewController owns, which performs actions in reaction to user input - gestures, text input, device movement, geographical location change etc.
+
+#### Interactor is: 
+- active component (has a lifecycle), 
+- final class (1),
+- can use other components for data retrieval etc.
+- should render methods on weakly held views.
+
+*(1) There is no good reason for inheritance of custom classes, ever.*
+
+#### ViewControllers:
+
+- can have many Interactors
+- should OWN interactors and see them via protocol,
+
+### Example
+
+**Step 1 - Create Interactor protocols:**
+
+```swift
+protocol PodBayDoorsInteracting: class {
+	func use(_ banner: BannerRendering)
+	func didTapMainButton()
+}
+```
+
+**Step 2 - Add Interactor implementation:**
+
+```swift
+final class PodBayDoorsInteractor {
+	// Why FU? We do want to crash if use() is not called before banner is used.
+	fileprivate weak var banner: BannerRendering! 
```

Comment:
instead of force unwrapping you could `fatalError` with described message, but it is probably just for a matter of example so you probably know it already 😝

Reply:
🤔 Good point actually.